src/main/java/org/interledger/setup/model/ReceiverQuery.java

```diff
@@ -0,0 +1,5 @@
+package org.interledger.setup.model;
+
+public interface ReceiverQuery {
+
```

Comment:
shouldnt we define the receiver query to contain the URI of the receiver endpoint, or do you forsee SPSP supporting alternative receiver endpoints?